net/message/message.go

```diff
@@ -176,8 +176,8 @@ func (m *NotifyETHLocked) Type() byte {
 
 // NotifyXMRLock is sent by XMRMaker to XMRTaker after locking his XMR.
 type NotifyXMRLock struct {
-	Address string     `json:"address" validate:"required"` // address the monero was sent to
-	TxID    types.Hash `json:"txID"`                        // Monero transaction ID (transaction hash in hex)
+	Address *mcrypto.Address `json:"address" validate:"required"` // address the monero was sent to
+	TxID    types.Hash       `json:"txID"`                        // Monero transaction ID (transaction hash in hex)
```

Comment:
this should be required

Reply:
I'll add that in a follow-up PR this week. I want to add required everywhere possible, but adding it often requires unit test fixes, so I didn't go too crazy in this PR.

Reply:
Here is the PR where the required tag will be added: https://github.com/AthanorLabs/atomic-swap/pull/330